src/Shell/Task/ExtractTask.php

```diff
@@ -308,6 +314,11 @@ protected function _extract()
         $this->_paths = $this->_files = $this->_storage = [];
         $this->_translations = $this->_tokens = [];
         $this->out();
+        if ($this->_countMarkerError) {
+            $this->err("{$this->_countMarkerError} marker error(s) detected.");
+            $this->err(" => Activate marker-error option to list errors.");
```

Comment:
Should `marker-error` have formatting to make it visually distinct? Or perhaps say `Use the --marker-error option to display errors`?

Reply:
Nice. I committed with phpcs errors fixed.